_pages/stormrider41.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  githubHandle: stormrider41
+  pageURL: stormrider41.github.io/stormrider41/
+  timestamp: TODAYS-DATE-2017/04/25
```

Comment:
@stormrider41 You're getting closer! The format should be `yyyy-mm-dd`, like `2017-04-25`.